TYPO3.Neos/Scripts/Gruntfile.js

```diff
@@ -172,6 +172,7 @@ module.exports = function (grunt) {
 						src = src.replace('data-toggle', 'data-neos-toggle');
 
 						// Tooltip
+						src = src.replace(/trigger: 'hover focus'/g, "trigger: 'hover'");
```

Comment:
This will disable tooltips opening on focus everywhere, not sure if that's desired. It's a bit of a grey area since I don't think anyone ever thought of this behavior, however just tested it and seems to work out fine.

However for the node tree it makes sense to disable if it causes issues.

Can be disabled specifically for the trees in https://github.com/neos/neos-development-collection/blob/master/Neos.Neos/Resources/Public/JavaScript/Content/Navigate/NodeTree.js#L229 + https://github.com/neos/neos-development-collection/blob/master/Neos.Neos/Resources/Public/JavaScript/Content/Navigate/ContextStructureTree.js#L172

Simply add `, trigger: 'hover'`